internal/backends/mcu/README.md

```diff
@@ -51,55 +53,53 @@ i_slint_backend_mcu = { git = "https://github.com/slint-ui/slint" }
 slint-build = { git = "https://github.com/slint-ui/slint" }
 ```
 
+## MCU-specific Setup
+
+Check the [MCU Setup](../../../docs/mcu_setup.md) guide for instructions on how to install
+the required tooling.
+
 ## Run the demo:
 
 ### The simulator
 
-
 ```sh
 cargo run -p printerdemo_mcu --features=mcu-simulator --release
 ```
 
 ### On the Raspberry Pi Pico
 
-You need nightly rust because that's the only way to get an allocator.
+#### Using probe-run
 
-Build the demo with:
+This require [probe-run](https://github.com/knurling-rs/probe-run) (`cargo install probe-run`)
+and to connect the pico via a probe (for example another pico running the probe)
+
+Then you can simply run with `cargo run`
 
 ```sh
-cargo +nightly build -p printerdemo_mcu --features=mcu-pico-st7789 --target=thumbv6m-none-eabi --release
+CARGO_TARGET_THUMBV6M_NONE_EABI_LINKER="flip-link" CARGO_TARGET_THUMBV6M_NONE_EABI_RUNNER="probe-run --chip RP2040" cargo +nightly run -p printerdemo_mcu --features=mcu-pico-st7789 --target=thumbv6m-none-eabi --release
 ```
 
-You should process the file with  [elf2uf2-rs](https://github.com/jonil/elf2uf2-rs)
+#### Without probe-run (Linux only!)
```

Comment:
why is it Linux only!?

Only the part with `udisksctl` is.  (I think)

Reply:
I have no idea whether that would work on Windows, so I was extra careful.